packages/react/jest.config.js

```diff
@@ -6,11 +6,13 @@ module.exports = {
     '!<rootDir>/**/index.(ts|tsx)',
     // do not collect from top level internal, PrimitiveCatalog, style and version files
     '!<rootDir>/src/(internal|PrimitiveCatalog|styles|version).(ts|tsx)',
+    // do not collect coverage for the Authenticator
+    '!<rootDir>/src/components/Authenticator/**/*.(ts|tsx)',
```

Comment:
Removing the `Authenticator` from coverage reporting as there is no plan to add more unit tests